lib/elixir/lib/task/supervised.ex

```diff
@@ -168,6 +169,7 @@ defmodule Task.Supervised do
       reducer: reducer,
       monitor_pid: monitor_pid,
       monitor_ref: monitor_ref,
+      ordered: ordered?,
```

Comment:
Okay. I'd still keep the internal map key as `:ordered?` but your choice @ericmj :)